eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -54,6 +54,7 @@ jobs:
     timeoutInMinutes: 240
 
     steps:
+      - script: |
```

Comment:
I think you want to undo the changes to this file now.

Reply:
done